lib/recurly/addon.php

```diff
@@ -18,6 +18,7 @@
  * @property string $revenue_schedule_type Optional field for setting a revenue schedule type. This will determine how revenue for the associated Plan should be recognized. When creating a Plan, if you supply an end_date and end_date available schedule types are never, evenly, at_range_start, or at_range_end.
  * @property DateTime $created_at The date and time the add-on was created.
  * @property DateTime $updated_at The date and time the add-on was last updated.
+ * @property string $plan_code Unique code to identify the plan.
```

Comment:
I wasn't trying to change them all around, but I figure if we do care about sorting them, then either alphabetically or the same order as the dev docs makes the most sense.

My only problem with matching the docs is that it could get out of date quickly if the docs change and this doesn't get updated. For that reason, I think maybe we should sort them alphabetically even if that's not the order of importance to the reader. The whole purpose of these is for the IDE to recognize these props, right? Not for humans reading the top of the file.

Reply:
I think this is a decision that should be made at a higher level and not in this PR. I've personally gone back and forth on how to sort attributes and it would be nice to decide on a strategy for that globally but I think it's too big to tackle here.